Description: Fix perl warning about deprecated "defined(%hash)"
 latex2html with current perl gives warnings about the deprecation of
 "defined(%hash)". This patches removes them by just removing the resp.
 keyword.
Author: Roland Stigge <stigge@antcom.de>

Index: latex2html-2008-debian1/latex2html.pin
===================================================================
--- latex2html-2008-debian1.orig/latex2html.pin	2010-07-31 16:13:21.000000000 +0200
+++ latex2html-2008-debian1/latex2html.pin	2012-04-09 17:21:52.000000000 +0200
@@ -532,7 +532,7 @@
 #
 # If possible, use icons of the same type as generated images
 #
-if ($IMAGE_TYPE && defined %{"icons_$IMAGE_TYPE"}) {
+if ($IMAGE_TYPE && %{"icons_$IMAGE_TYPE"}) {
     %icons = %{"icons_$IMAGE_TYPE"};
 }
 
@@ -2112,7 +2112,7 @@
 		"\nCould not find translation function for $default_language.\n\n")
 	    }
 	);
-	if ($USE_UTF ||(!$NO_UTF &&(defined %unicode_table)&&length(%unicode_table)>2)) {
+	if ($USE_UTF ||(!$NO_UTF &&(%unicode_table)&&length(%unicode_table)>2)) {
 	    &convert_to_unicode($_)};
     }
     $_ = join('', @case_processed, $_); undef(@case_processed);
@@ -9117,7 +9117,7 @@
     my ($charset) = "${CHARSET}_character_map_inv";
     $charset =~ s/-/_/g;
     # convert upper 8-bit characters
-    if (defined %$charset &&($CHARSET =~ /8859[_\-]1$/)) {
+    if (%$charset &&($CHARSET =~ /8859[_\-]1$/)) {
 	s/([\200-\377])/
 	    $tmp = $$charset{'&#'.ord($1).';'};
 	    &mark_string($tmp) if ($tmp =~ m!\{!);
